nessie/models/student_schema.py

```diff
@@ -64,7 +64,7 @@ def refresh_from_staging(table, term_id, sids, transaction, truncate_staging=Tru
     # matching those restrictions. If there are no restrictions, the entire destination table can be truncated.
     refresh_conditions = []
     refresh_params = []
-    if (term_id and table == 'student_enrollment_terms'):
+    if term_id:
```

Comment:
@raydavis Will removing the `table == 'student_enrollment_terms'` cause a problem when this method is called for the student profile tables?  In `GenerateMergedStudentFeeds.generate_feeds`, we call `student_schema.refresh_all_from_staging(profile_tables)` where `profile_tables` includes student_profiles, student_academic_status, etc. - tables that don't have a `term_id` column.

Reply:
I think I just answered my own question. `term_id` will be None in those cases, so the `term_id` condition won't get appended.